rust/processor/src/models/object_models/v2_objects.rs

```diff
@@ -278,14 +269,20 @@ impl Object {
                         is_token: res.is_token,
                         is_fungible_asset: res.is_fungible_asset,
                         is_deleted: res.is_deleted,
-                    })
+                    };
                 },
-                Err(_) => {
+                Err(e) => {
                     std::thread::sleep(std::time::Duration::from_millis(QUERY_RETRY_DELAY_MS));
+                    warn!(
+                        transaction_version,
+                        error = ?e,
+                        "Failed to get object from current_objects table for object_address: {}, retrying in {} ms. If the object doesn't exist you should probably backfill db.",
```

Comment:
if db doesn't setup correctly, retry forever may not help; consider adding a upper bound for retrying.

Reply:
what do you mean by not setup correctly?

Reply:
db requires backfilling

Reply:
that's a good point. the new columns require backfill or else the token v2 and fungible asset processors will not advance. i guess the question is whether we should 
1. retry forever. this is a silent error and the processor will be stuck. 
2. return an error after a couple retries to signal users they need to backfill 

i think option 2 is better so users know they need to take an action to advance the processors.